### PR TITLE
[autotools] Add missing APP_DATA_DIR in configure.ac

### PR DESCRIPTION
**Pull request by hudokkow**
@MilhouseVH, please test if it solves [this](https://github.com/xbmc/xbmc/pull/10359#discussion_r77879558).

Done blindly. Don't want to play with autotools now.

### CODE REVIEW

**Comment by MilhouseVH** [comment]
Many thanks, that's fixed it!

```
if(NOT KODI_DATA_DIR)
  set(KODI_DATA_DIR /usr/share/kodi)
endif()
```

**Comment by hudokkow** [comment]
@wsnipex for sanity check.

File: configure.ac [hunk 1]
```diff
@@ -2238,9 +2238,11 @@ fi
 eval "APP_PREFIX=${prefix}"
 eval "APP_LIB_DIR=${libdir}/${APP_NAME_LC}"
 eval "APP_INCLUDE_DIR=${includedir}/${APP_NAME_LC}"
+eval "APP_DATA_DIR=${prefix}/share/${APP_NAME_LC}"
```
Comment (wsnipex) [inline configure.ac]: ${datarootdir}/${APP_NAME_LC}

**Comment by hudokkow** [comment]
Updated!

**Pull request merged**